### PR TITLE
Follow up fixes

### PR DESCRIPTION
**Pull request by ap-justin**
## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/components/landing/BenefitssCrousal.jsx [hunk 1]
```diff
@@ -67,7 +61,7 @@ const BenefitsCarousel = ({ slides }) => {
         }}
         onSlideChange={(swiper) => handleSlideChange(swiper)}
         modules={[EffectCoverflow, Pagination, Navigation]}
-        className="benefits_Carousel"
+        className="benefits_crousal"
```
Comment (SovereignAndrey) [inline src/components/landing/BenefitssCrousal.jsx:64]: @ap-justin Another carousel spelling fix I missed. The css class will need updating too, please. 
```suggestion
        className="benefits_carousel"
```

File: src/components/landing/BenefitssCrousal.jsx [hunk 2]
```diff
@@ -131,4 +125,4 @@ const BenefitsCarousel = ({ slides }) => {
   );
 };
 
-export default BenefitsCarousel;
+export default BenefitssCrousal;
```
Comment (SovereignAndrey) [inline src/components/landing/BenefitssCrousal.jsx:128]: We shouldn't revert these spelling fixes that were done in the PR branch.

**Review by SovereignAndrey — CHANGES_REQUESTED** [review]
Looks good. Just need to not revert the already made spelling fixes.

File: src/components/landing/BenefitssCrousal.jsx (hunk 1 above)
Comment (ap-justin) [reply src/components/landing/BenefitssCrousal.jsx:64]: I would revert to the renamed file, got conflict when I changed the unrenamed file

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**